joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstSummaryVisitor.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Using
 
 trait AstSummaryVisitor(implicit withSchemaValidation: ValidationMode) { this: AstCreator =>
 
-  def summarize(): RubyProgramSummary = {
+  def summarize(dependencyDownloadPass: Boolean = false): RubyProgramSummary = {
```

Comment:
Might make for clearer consistent language as the CPG has the whole `isExternal` property
```suggestion
  def summarize(asExternal: Boolean = false): RubyProgramSummary = {
```